pandas/io/sql.py

```diff
@@ -585,7 +585,8 @@ def read_sql(
     ----------
     sql : str or SQLAlchemy Selectable (select or text object)
         SQL query to be executed or a table name.
-    con : SQLAlchemy connectable, str, or sqlite3 connection
+    con : AdbcConnection, SQLAlchemy connectable, str, or sqlite3 connection
```

Comment:
Strictly speaking it is not an `adbc_driver_manager.AdbcConnection` but `adbc_driver_manager.dbapi.Connection`, so maybe would use a more generic "ADBC connection" ?

Also, a few lines below, it says "user is responsible for engine disposal and connection closure for the SQLAlchemy connectable" -> ".. for the ADBC connection and SQLAlchemy connectable"

Reply:
Hmm strange. This is available at the top of the package though:

```python
>>> import adbc_driver_manager
>>> adbc_driver_manager.AdbcConnection
<class 'adbc_driver_manager._lib.AdbcConnection'>
```

Reply:
Yes, that object exists, but what I meant is that we a user needs to pass here is not that object. The result of dbapi.connect() is not a `adbc_driver_manager.AdbcConnection` but a `adbc_driver_manager.dbapi.Connection`

Reply:
This should be updated now - thanks for the feedback